### DISCUSSION TITLE
Open API definition

### DISCUSSION THREAD

**Discussion by fcostasilva** [tidal-music/discussions#22] [Ideas & Feature requests]
Hi Guys, 

Regarding the Tidal API, Is the OpenAPI definition file available somewhere? I couldn't find it in the documentation. It would be useful to be available at the API reference page.

**Comment by stefanmoro** [comment]
Hey @fcostasilva, thanks for the input!

This is something we have discussed internally, so really good to know there is interest in having that available. 

We'll see what we can do about it!

**Reply by fcostasilva** [reply]
Thanks for considering this feature. The definition is useful in many ways, from importing it to an API platform like Postman to generate SDKs. I hope it is integrated into the docs at some point.

**Comment by jozefKruszynski** [comment]
Also would love to be able to get the openapi definition.
It would make integration much simpler.

**Comment by aliaksei-taliuk** [comment]
@fcostasilva @jozefKruszynski Hi! Thanks for your patience!

Happy to announce that we added this feature and you can download the underlying specs directly from the [API Reference](https://developer.tidal.com/apiref) You can find the link down below the list of available API endpoints:
<img width="277" alt="image" src="https://github.com/tidal-music/discussions/assets/95020987/5a208340-caea-4163-9e3f-ae4669f02a8d">

**Reply by jozefKruszynski** [reply]
This is great
